src/EditorFeatures/CSharpTest/QuickInfo/DiagnosticAnalyzerQuickInfoSourceTests.cs

```diff
@@ -32,9 +32,11 @@ public async Task ErrorTitleIsShownOnDisablePragma()
     {
         await TestInMethodAsync(
             """
+
             #pragma warning disable CS0219$$
```

Comment:
It doesn't look like there are meaningful changes in this file. Consider excluding it from the PR

Reply:
These are needed. `TestInMethod` injects the code into a method but without newlines between the opening/closing braces of the method, so the `#pragma` was in trailing position and the test stopped working.